pytests/iroha_cli_tests/pyproject.toml

```diff
@@ -8,6 +8,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.11"
 faker = "*"
+cffi = ">=1.17.0"
```

Comment:
Why did you need to add this package? Was it to force an update to newer version?

(If that's the case, `poetry update` would've done it, no need to add a transitive dependency into `pyproject.toml`)

Reply:
Just running poetry update will also get you `1.17.1` and put this update in the lockfile. No need to put this dep into `pyproject.toml`

Reply:
It looks like this line, indeed, just updated the direct dependencies.